recipes/pyorerun/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Step 1: Install dependencies from Conda Forge
+conda install -c conda-forge ezc3d trimesh rerun-sdk biorbd numpy -y
```

Comment:
You should not be installing anything manually with conda here. All dependencies should be listed in the recipe.

This line can probably be removed without replacement

Reply:
Done